w3c.json

```diff
@@ -0,0 +1,9 @@
+{
+  "group": 115198,
+  "contacts": [
+    "tidoust",
+    "jernoble",
+    "mounirlamouri"
```

Comment:
Might add me here. I've picked up most editor work from Mounir.

Reply:
I'm all in favour for you to be added as editor but we may want to do this in a follow-up PR, just to keep the history clean.

As a side note, I am not sure that the editors are listed in this file but Francois would know :)

Reply:
Right, adding you to the list is fine, but this isn't meant to be the list of editors, rather contact points for administrative requests on the repo (should there ever be administrative requests):
https://w3c.github.io/w3c.json.html

Group chairs and team contact are usually a good starting point.

Reply:
Thanks for clarifying. List looks fine then.